generic/tests/cfg/timedrift.cfg

```diff
@@ -77,25 +64,37 @@
             test_duration = 600
             smp = 2
             stop_time = 60
-            drift_threshold = 10
         - with_signal_stop:
-            # starting from Win2012 Windows no longer relies on number of injected RTC interrupts for time keeping,
-            # and the time drift is seen because "stop" monitor command also stops PM timer,
-            # so this is unsupported scenario from Win2012.
-            Windows:
-                only Win2000, WinXP, Win2003, WinVista, Win7, Win2008
             no RHEL.3.9, RHEL.4
             type = timedrift_with_stop
             stop_interations = 1
             stop_with_signal = yes
             stop_time = 60
-            drift_threshold = 10
-            drift_threshold_single = 3
-            variants:
-                - clock_host:
-                    rtc_clock = host
-                - clock_vm:
-                    rtc_clock = vm
+    variants:
+        - default_clock:
```

Comment:
For windows, `default_clock` is duplicate with `clock_host`,  and for linux, it is duplicate with `clock_vm`. 
Need be improve again.